.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -12,7 +12,11 @@ If you think you have found a potential security issue, please do not post it as
 -->
 
 **What happened**:
-<!-- include log lines if possible -->
+<!--
```

Comment:
@jayanthvn can you update PR template as well?

Reply:
Sure Balaji will do.